docs/static/spec/rasa.yml

```diff
@@ -464,6 +464,18 @@ paths:
           type: boolean
           default: False
           description: Force a model training even if the data has not changed
+      - in: query
```

Comment:
do you also plan on adding the other parameters or would that be something for follow-up PRs?

Reply:
I just extract the parameters we use in training CLI here https://github.com/RasaHQ/rasa/blob/main/rasa/cli/train.py#L211:L228
I'm not sure if there're other parameters, if so I could add them to this PR too

Reply:
Up to you. We can also do that in followup PRs so that this one is already in.

Reply:
Yes, it's up to the parameters Rasa support in training